README.md

```diff
@@ -0,0 +1,36 @@
+# Gatekeeper
+
+This application is responsible for opening and closing the front gate.
+
+## Context
+
+Gatekeeper is a raspberry pi (hostname `zuul`) that is responsible for opening the front gate for guest and members. The pi is used to control the gate remote control and uses a Sim800 module to receive calls
+
+## Usage
+
+The gatekeeper application uses a whitelist in the following format:
+
+    +123456 owner
+
+The first element is the phone number in international notation without spaces or other punctuation, the second element is a free text field describing the owner of the phone number.
```

Comment:
the first 4 characters of the free text field may be used on the flappers (though that currently doesn't work)